app/graphql/resolvers/news_items_search.rb

```diff
@@ -50,7 +51,11 @@ def apply_data_provider(scope, value)
   end
 
   def apply_data_provider_id(scope, value)
-    scope.joins(:data_provider).where(data_providers: { id: value })
+    scope.where(data_provider_id: value)
+  end
+
+  def apply_exclude_data_provider_ids(scope, value)
+    scope.exclude_data_providers(value)
```

Comment:
why is the one `where` for a single id above here in this file and this for multiple ids (`where.not(data_provider_id: data_provider_ids)`) placed in the model?

Reply:
Solved in https://github.com/ikuseiGmbH/smart-village-app-mainserver/commit/444c7559fc08f7ab7a3cb7d777551753d812ea5b